audit-appendix.html

```diff
@@ -75,21 +81,20 @@ <h3>Ingest</h3>
             <td>The number of bytes ingested for the filegroup or the file</td>
           </tr>
           <tr>
-            <td>files ingested</td>
+            <td>Files Ingested</td>
             <td>The number of files ingested</td>
           </tr>
         </tbody>
       </table>
 
       <p class="note">
-        This could be split into two discrete events, one for the filegroup and one for each file.
+        This could be split into multiple discrete events, one for the filegroup and one for each file in the filegroup.
```

Comment:
The intention of this was to say that there could be a separate Audit event for each type of ingestion. So, potentially a `Filegroup Ingest` and `File Ingest`. I'm not sure how important it is to call out, but that is how we handle the ingestion events in ACE so I added it as a note.

Reply:
If a group of 5 files that are part of a file group are ingested into ACE, does that create 2 events (one for the file group, one for all the files) or 6 events (one for the file group, one for each of the 5 files)? I was assuming the latter when I made the change.

Reply:
I realized I was conflating ACE's events for starting an audit with the registration of a filegroup, so for that example only 5 ingest events will be created.

I'm still not sure if we really need to call this out anymore, I'm on the fence about it. Especially since earlier in the doc we call out that this is preliminary, so I don't know if commends/notes like that are necessary.

Reply:
Agreed, I think this note could be left out. How it is implemented will depend on the internals of the DDP. It's not really necessary to call out these potential variations.